admin-part/src/main/kotlin/Coverage.kt

```diff
@@ -162,21 +164,31 @@ private fun Sequence<Session>.bundlesByTests(
     context: CoverContext,
     cache: AtomicCache<TypedTest, BundleCounter>?
 ): Map<TypedTest, BundleCounter> = takeIf { it.any() }?.run {
+    val allTests = allTests(cache)
     groupBy(Session::testType).map { (testType, sessions) ->
         sessions.asSequence().flatten()
             .groupBy { TypedTest(it.testName, testType) }
             .filterNot { cache?.map?.containsKey(it.key) ?: false }
-            .map { it.key to it.value.asSequence().bundle(context) }
-            .toMap(mutableMapOf())
+            .mapValuesTo(allTests) { it.value.asSequence().bundle(context) }
     }.reduce { m1, m2 ->
         m1.apply { putAll(m2) }
     }.let { mutableMap ->
-        val finalizedTests = flatMap { map -> map.testStats.keys }
+        val finalizedTests = flatMap { it.testStats.keys }
         val testsAddToCache = mutableMap.filterKeys { finalizedTests.contains(it) }
         mutableMap + (cache?.putAll(testsAddToCache) ?: emptyMap())
     }
 } ?: emptyMap()
 
+private fun Sequence<Session>.allTests(
+    cache: AtomicCache<TypedTest, BundleCounter>?
```

Comment:
not well naming because we have a filter here and our goal to set the default bundle for tests
maybe it should move BundleCounter("") to default parameter
```suggestion
    cache: AtomicCache<TypedTest, BundleCounter>?,
    empty: BundleCounter = BundleCounter("")
```